alert-common/src/main/java/com/synopsys/integration/alert/common/event/DistributionEvent.java

```diff
@@ -24,30 +24,43 @@
 
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.jetbrains.annotations.Nullable;
+
 import com.synopsys.integration.alert.common.message.model.MessageContentGroup;
-import com.synopsys.integration.alert.common.persistence.accessor.FieldUtility;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
+import com.synopsys.integration.alert.common.persistence.model.job.DistributionJobModel;
 
 public class DistributionEvent extends ContentEvent {
-    private static final long serialVersionUID = -2067819359358348281L;
-    private final FieldUtility fieldUtility;
-    private final String configId;
+    private static final long serialVersionUID = -7858733753649257748L;
```

Comment:
I would love to know if this is required, or if we can remove it.

Reply:
Are they compiler warnings or IDE warnings? I could have sworn we determined that we should no longer be generating `serialVersionUID` on `Serializable` classes.